advanced_topics.qmd

```diff
@@ -1344,7 +1345,7 @@ plot + facet_wrap(vars(language), ncol = 4)
 #| label: pums-langauge-northeast-plot-pdf
 #| echo: false
 #| fig-height: 9
-#| fig-cap: "Proportion of speakers by language in the Northeast by PUMA for English, French, Portuguese, and Spanish"
+#| fig-cap: "Proportion of speakers whose primary language is not English by language in the Northeast by PUMA for Chinese, French, Portuguese, and Spanish"
```

Comment:
Same idea as above. "whose primary language is not English by language in the Northeast by PUMA for" just has too many prepositions, and saying "language" twice close together just felt extra awkward.

Reply:
```suggestion
#| fig-cap: "Proportion of speakers whose primary language is not English in the Northeast by PUMA for Chinese, French, Portuguese, and Spanish"
```

Like this?

Reply:
Yeah, better. But maybe we can drop the list of languages too? They're in the plot.

Reply:
Done (sorry for re-writting history on you there)

Reply:
I made another edit (added a comma). WDYT?

Reply:
```suggestion
#| fig-cap: "Proportion of speakers whose primary language is not English in the Northeast, by PUMA"
```